nav2_map_server/src/map_server/main_3d.cpp

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2018 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <memory>
+#include <string>
+
+#include "nav2_map_server/map_server.hpp"
+#include "sensor_msgs/msg/point_cloud2.hpp"
+
+#include "rclcpp/rclcpp.hpp"
+
+int main(int argc, char ** argv)
+{
+  std::string node_name("map_server");
+
+  rclcpp::init(argc, argv);
+  auto node = std::make_shared<nav2_map_server::MapServer<sensor_msgs::msg::PointCloud2>>();
```

Comment:
Semantically, I don't really see why this shouldn't be `MapServer3D` without template arguments using the map server in the 2d directory named as such with the `template <>` removed. The actual object didn't make use of the template argument so this seems like TMI and more confusing to a later reader.